frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js

```diff
@@ -511,7 +511,30 @@ class DataCube {
   }
 
   hasDataAtPositionAndZoomStep(voxel: Vector3, zoomStep: number = 0) {
-    return this.getBucket(this.positionToZoomedAddress(voxel, zoomStep)).hasData();
+    const bucket = this.getBucket(this.positionToZoomedAddress(voxel, zoomStep));
+
+    if (!(bucket instanceof DataBucket)) {
+      // This is a NullBucket (e.g., because it's out of the bounding box or in a not-existing
+      // magnification). Don't assume fallback rendering.
+      return true;
```

Comment:
I can't figure out why `true` is returned, here. If the magnification is not existing, shouldn't `hasDataAtPositionAndZoomStep` return `false` for this position and zoomStep?

Reply:
Ah, I forgot to rename this method. The semantics of it should rather be "does the given zoomstep for the given position yield 'useful' data or should we look at a fallback resolution". For nullbuckets, looking at the fallback resolutions won't have any benefit, which is why `true` is returned. I'll definitely add a comment to this method, but another name should also be used. Maybe `isVoxelAtZoomStepRenderable`?

Reply:
I went for the name `isZoomStepRenderableForVoxel` and improved the comments in https://github.com/scalableminds/webknossos/pull/5519/commits/b82ccf510a119d71314022110566833efe1f7866.